rskj-core/src/main/java/co/rsk/peg/bitcoin/FlyoverRedeemScriptBuilder.java

```diff
@@ -0,0 +1,8 @@
+package co.rsk.peg.bitcoin;
+
+import co.rsk.bitcoinj.script.Script;
+import co.rsk.crypto.Keccak256;
+
+public interface FlyoverRedeemScriptBuilder {
```

Comment:
Why merge it into the `master` branch instead of the integration branch, `wip/fed-scripts-refactors-integration`?